content/docs/troubleshooting/_index.md

```diff
@@ -1,8 +1,8 @@
 ---
 title: Troubleshooting & Support
```

Comment:
Since this page is now under "SUPPORT" in the TOC, we should consider renaming the title to just "Troubleshooting".